src/Radicle/Internal/PrimFns.hs

```diff
@@ -668,8 +671,8 @@ readValues
     => Text
     -> Text
     -> m [Value]
-readValues name s = do
-    let p = parseValues ("[" <> name <> "-primop]") s
+readValues label s = do
+    let p = parseValues ("[" <> label <> "]") s
```

Comment:
This conflicts with #300. I suggest to wrap the label in square brackets at the call site, i.e. the definition of the primitives.